platforms/xamarin.ios/System.Runtime.Serialization.csproj

```diff
@@ -2,6 +2,7 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
+    <AssemblyOriginatorKeyFile>../silverlight.pub</AssemblyOriginatorKeyFile>
```

Comment:
I would suggest putting this in the dir.props and condition it on "UseSilverlightKey=true"